### PR TITLE
Adding virtual implementations for remaining HappyBase Table methods.

### PR DESCRIPTION
**Pull request by dhermes**
@jonparrott I figured this would be good for you to get a sense of why I'd be sending PRs for helpers before actually implementing the methods themselves. The [helpers][1] themselves contain quite a bit of code (HappyBase is a nice abstraction and the work is worth it).

[1]: https://github.com/dhermes/gcloud-python-bigtable/blob/b2c61ee0d105013e4152711c8563cc3365ea58d7/gcloud_bigtable/happybase/table.py#L97-L334

### CODE REVIEW

File: gcloud/bigtable/happybase/connection.py [hunk 1]
```diff
@@ -97,7 +97,7 @@ class Connection(object):
 
     The arguments ``host``, ``port``, ``compat``, ``transport`` and
     ``protocol`` are allowed (as keyword arguments) for compatibility with
-    HappyBase. However, they will not be used in anyway, and will cause a
+    HappyBase. However, they will not be used in any way, and will cause a
```
Comment (dhermes) [inline gcloud/bigtable/happybase/connection.py]: Also I squeezed in a docstring fix that I noticed when copy-ing it.

Comment (jonparrott) [inline gcloud/bigtable/happybase/connection.py]: Niice.

File: gcloud/bigtable/happybase/table.py [hunk 2]
```diff
@@ -23,6 +23,7 @@
 from gcloud.bigtable.column_family import GCRuleIntersection
 from gcloud.bigtable.column_family import MaxAgeGCRule
 from gcloud.bigtable.column_family import MaxVersionsGCRule
+from gcloud.bigtable.happybase.batch import _WAL_SENTINEL
```
Comment (jonparrott) [inline gcloud/bigtable/happybase/table.py]: I'm not sure how I feel about importing private variables from other modules.

**Comment by jonparrott** [comment]
Cool, this is helpful. LGTM, but with minor concern about that import.

File: gcloud/bigtable/happybase/table.py (hunk 2 above)
Comment (dhermes) [inline gcloud/bigtable/happybase/table.py]: Ditto. I almost ripped it out when porting from `gcloud-python-bigtable` but then I saw why it's done: the default `wal` is passed down into `batch` helpers.

I plan on ripping this out and using `**kwargs` with a warning as is done in the `Connection` constructor and as is done here in `Table.scan`.

Comment (jonparrott) [inline gcloud/bigtable/happybase/table.py]: Okay, fair enough.

**Pull request merged**